resources/event-sources/templates/100-pre-upgrade-job.yaml

```diff
@@ -0,0 +1,136 @@
+---
```

Comment:
General comments:
- Let's add the following templating everywhere:
```
name: {{ template "controller-manager.fullname" . }}-pre-upgrade-helm
```
- namespace is not required
- annotation: `helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded` needs to be added

Reply:
My intention was to have the same code in both "config/" and "templates/" so that the job works with "ko" too. Cause this job will be removed in the next release, I made it as  simple as possible. From this perspective, making "name" configurable can be left out. Is also much simpler if something is happen on the customer side, to analyze the issue, I meant.
I've added the ```helm.sh/hook-delete-policy```.

Reply:
Again, it not clear to me the value of `ko` here.
Templating is considered one of the best practices of Helm. So we should do it.
Namespaces are redundant in the chart as kyma-installer take care of it.

Reply:
See my early comment. The required changes are done.